test/e2e/handler/rollback_test.go

```diff
@@ -47,6 +105,10 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 		It("should rollback failed state configuration", func() {
 			updateDesiredState(linuxBrUpNoPorts(bridge1))
+
+			By("Should not be available") // Fail fast
+			policyConditionsStatusConsistently().ShouldNot(containPolicyAvailable())
```

Comment:
shouldn't we check this again after `waitForDegradedTestPolicy` is done?

Reply:
We are degraded we are not going back to Available